src/test/tst_StyleMatchTree.cpp

```diff
@@ -298,7 +298,8 @@ TEST_CASE("Inherit properties from all matching selectors with parent", "[match]
   REQUIRE("3" == propertyAsString(pm, "propC"));
 }
 
-TEST_CASE("Inherit properties from all matching selectors with classed parents", "[match]")
+TEST_CASE("Inherit properties from all matching selectors with classed parents",
+          "[match]")
```

Comment:
@gck-ableton: this change (and the last 3 in `src/test/tst_CssParser.cpp`) could be extracted into a `clang-format` commit.